invokeai/app/api/routers/board_images.py

```diff
@@ -24,11 +26,14 @@ async def create_board_image(
 ):
     """Creates a board_image"""
     try:
-        result = ApiDependencies.invoker.services.board_images.add_image_to_board(board_id=board_id, image_name=image_name)
+        result = ApiDependencies.invoker.services.board_images.add_image_to_board(
```

Comment:
How would you feel about changing all `board_id` arguments to `Path`? That would help us have a consistent thing to key off of for authorization. It also adds some complexity now that we are _only_ sending in image names for the delete functionality. We will have to check access for each image vs checking for board access. I think you removed the board_id bc its a redundant for as long as we have a one-to-many relationship but it does make authorization more complex